config/initializers/rollbar.rb

```diff
@@ -0,0 +1,71 @@
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
have you provided rollbar token to heroku?

Reply:
yes